release_notes.md

```diff
@@ -267,141 +271,161 @@ This release is not required any additional steps to use.
       ```
 
-      **Dependencies list** 
+   **Dependencies list**
 
-      | dependency name | provider version |
-      |:--------:|:-------:|
-      |   RDP5   |    5    |
-      |   RDP6   |    6    |
+| dependency name | provider version |
+|:---------------:|:----------------:|
+|      RDP5       |        5         |
+|      RDP6       |        6         |
 
-      **Example**
+**Example**
 
-      ```
-      pip install th2-data-services[rdp5]
-      ```
+   ```
+   pip install th2-data-services[rdp5]
+   ```
 
 ## Features
+
 1. [TH2-4289] Data.build_cache and Data.from_cache_file features were added.
 2. Added `Data.cache_status` property
 
 ## Improvements
+
 1. [TH2-4379] Speed improvements in json deserialization.
-   - StreamingSSEAdapter will now handle bytes from sse-stream into Dict objects.
-   - SSEAdapter is now deprecated class.
+    - StreamingSSEAdapter will now handle bytes from sse-stream into Dict objects.
+    - SSEAdapter is now deprecated class.
 2. Data object will generate a warning if you put to it an object that has generator type.
 
 ## BugFixes
-1. [TH2-4385] Logging in Data object slows down the ds library very much. 
-   - Logging was removed.
-   - `add_stderr_logger` and  `add_file_logger` are not available anymore.
+
+1. [TH2-4385] Logging in Data object slows down the ds library very much.
+    - Logging was removed.
+    - `add_stderr_logger` and  `add_file_logger` are not available anymore.
 2. [TH2-4380] Fixed apply_adpater feature for GetMessages / GetEvents / GetEventById / GetMessageById
 3. [TH2-3767] Fixed bug with limit of Data object in Windows.
 4. [TH2-4460] Fixed bug where GRPC omitted fields with None value in response.
 
 # v2.0.0
 
 ## User impact and migration instructions
-By installing the package you will no longer get RDP package. 
+
+By installing the package you will no longer get RDP package.
 If you want to use RDP you have to specify dependency in square brackets `[ ]`
 
 1. [I] Adapter interface got required handle_stream method.
    [M] Implement new method for your adapters.
 
 ## Features
+
 1. [TH2-4128] pip no longer installs RDP by default
 2. [TH2-4128] extra dependencies can be installed using square brackets after package name.
     - Example: `pip install th2-data-services[lwdp]`
-    
+
    Available data sources implementations:
 
-   | provider | version                            |
-----------|:--------:|------------------------------------|
-   | lwdp     | latest version of lwdp             |
-   | lwdp1    | latest version of lwdp (version 1) |
-   | lwdp2    | latest version of lwdp (version 2) |
+| provider |              version               |
+|----------|:----------------------------------:|
+| lwdp     |       latest version of lwdp       |
+| lwdp1    | latest version of lwdp (version 1) |
+| lwdp2    | latest version of lwdp (version 2) |
 
 3. [TH2-4493] Adapter interface got handle_stream method.
+4. [TH2-4490] Added `map_stream` method to Data. 
```

Comment:
I hope, you agree that it's not all. map_stream also can take Adapters, not only map function.

We also have to add some examples to our demo example script